exporter/exporterparser/exparser.go

```diff
@@ -32,7 +32,32 @@ import (
 	tracetranslator "github.com/census-instrumentation/opencensus-service/translator/trace"
 )
 
-func exportSpans(ctx context.Context, exporterName string, te trace.Exporter, td data.TraceData) error {
+// ocProtoSpansToOCSpanDataInstrumented converts
+// OpenCensus Proto TraceData to OpenCensus-Go SpanData.
+// The "Instrumented" suffix serves to document that this
+// function is traced but also has stats for self-observability.
+//
+// This is a bootstrapping mechanism for us to re-use as many of
+// the OpenCensus-Go trace.SpanData exporters which were written
+// by various vendors and contributors. Eventually the goal is to
+// get those exporters converted to directly receive
+// OpenCensus Proto TraceData.
+func ocProtoSpansToOCSpanDataInstrumented(ctx context.Context, exporterName string, te trace.Exporter, td data.TraceData) (aerr error) {
+	ctx, span := trace.StartSpan(ctx,
```

Comment:
q: what are the plans for these spans? iirc no trace exporter is setup and for now they are never sampled.

Reply:
zPages use them